src/mappings/_utils.ts

```diff
@@ -8,6 +8,10 @@ import { VaultId as VaultIdV1021000 } from "../types/v1021000";
 import { encodeLegacyVaultId, encodeVaultId } from "./encoding";
 import { ApiPromise, WsProvider } from '@polkadot/api';
 import * as process from "process";
+import { CurrencyExt, CurrencyIdentifier, currencyIdToMonetaryCurrency, newMonetaryAmount } from "@interlay/interbtc-api";
+import { BigDecimal } from "@subsquid/big-decimal";
+import { getInterBtcApi } from "../processor";
+
```

Comment:
nitpick (not blocking): an extra blank line slipped in here

Reply:
FIxed!